modules/administration-guide/pages/installing-che-in-an-oidc-unsupported-kubernetes.adoc

```diff
@@ -0,0 +1,220 @@
+:_content-type: PROCEDURE
+:navtitle: Installing {prod-short} in an OIDC Unsupported {kubernetes}
+:description: Installing {prod-short} in {kubernetes} that doesn't support OIDC yet.
+:keywords: installing-che-in-an-unsupported-{kubernetes}
+:page-aliases: installation-guide:installing-che-in-oidc-unsupported-kubernetes
+
+[id="installing-{prod-id-short}-in-odic-unsupported-kubernetes"]
+= Installing {prod-short} in OIDC Unsupported {kubernetes} Cluster
+
+You can deploy {prod-short} into a {kubernetes} cluster that doesn't support OIDC.
+
+WARNING: vCluster is a third party tool. See link:https://www.vcluster.com/docs/what-are-virtual-clusters[Introduction to vCluster]
+
+.Prerequisites
+
+* Helm CLI. See link:https://helm.sh/docs/intro/install/[Installing Helm].
+
+* vCluster CLI. See link:https://www.vcluster.com/docs/getting-started/setup[Installing vCluster CLI].
+
+* `{orch-cli}` stable release with `oidc-login` plugin. See link:https://kubernetes.io/docs/tasks/tools/#kubectl[Installing `{orch-cli}`], link:https://github.com/int128/kubelogin[Installing `kubelogin`].
```

Comment:
Should `kubelogin` be installed separately or kubectl is enough? 
I ask because of this:
```
% kubectl krew install oidc-login
error: unknown command "krew" for "kubectl"
```
```
% kubectl version
Client Version: v1.28.4
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: v1.26.1
WARNING: version difference between client (1.28) and server (1.26) exceeds the supported minor version skew of +/-1
```